### PR TITLE
Hide dashboard overview and make dashboards management available

### PR DESCRIPTION
**Pull request by yubonluo**
### Description

When users create a new workspace, they can not choose dashboard overview and can choose dashboards management.

## Screenshot
image

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/opensearch_dashboards_overview/public/plugin.ts [hunk 1]
```diff
@@ -81,9 +81,17 @@ export class OpenSearchDashboardsOverviewPlugin
       map((hasOpenSearchDashboardsApp) => {
         return () => {
           if (!hasOpenSearchDashboardsApp) {
-            return { status: AppStatus.inaccessible, navLinkStatus: AppNavLinkStatus.hidden };
+            return {
+              status: AppStatus.inaccessible,
+              navLinkStatus: AppNavLinkStatus.hidden,
+              workspaceAccessibility: WorkspaceAccessibility.NO,
```
Comment (SuZhou-Joe) [inline src/plugins/opensearch_dashboards_overview/public/plugin.ts:87]: Change on registeration is enough I think.

File: src/plugins/workspace/public/utils.ts [hunk 2]
```diff
@@ -70,20 +70,29 @@ export const featureMatchesConfig = (featureConfigs: string[]) => ({
...
-      navLinkStatus !== AppNavLinkStatus.hidden &&
-      !chromeless &&
-      category?.id !== DEFAULT_APP_CATEGORIES.management.id
+    ({ navLinkStatus, chromeless, category, workspaceAccessibility, id }) => {
+      const filterCondition =
+        navLinkStatus !== AppNavLinkStatus.hidden &&
+        !chromeless &&
+        workspaceAccessibility !== WorkspaceAccessibility.NO;
+      if (category?.id === DEFAULT_APP_CATEGORIES.management.id) {
+        return filterCondition && id === 'management';
```
Comment (SuZhou-Joe) [inline src/plugins/workspace/public/utils.ts:83]: Could you please add some comment here to elaborate?

File: src/plugins/workspace/public/utils.ts [hunk 3]
```diff
@@ -70,20 +70,29 @@ export const featureMatchesConfig = (featureConfigs: string[]) => ({
...
+      } else {
+        return filterCondition;
+      }
```
Comment (SuZhou-Joe) [inline src/plugins/workspace/public/utils.ts:86]:
```suggestion

      return filterCondition;
```
Nit: better to remove useless `else`.

File: src/plugins/opensearch_dashboards_overview/public/plugin.ts (hunk 1 above)
Comment (yubonluo) [reply src/plugins/opensearch_dashboards_overview/public/plugin.ts]: Have deleted the useless code.

File: src/plugins/workspace/public/utils.ts (hunk 2 above)
Comment (yubonluo) [reply src/plugins/workspace/public/utils.ts:82]: The comment has been added

File: src/plugins/workspace/public/utils.ts (hunk 3 above)
Comment (yubonluo) [reply src/plugins/workspace/public/utils.ts]: Have deleted `else`

**Review by ruanyl — APPROVED** [review]

**Comment by ruanyl** [comment]
@yubonluo do we still need to get this PR merged?

**Comment by yubonluo** [comment]
> @yubonluo do we still need to get this PR merged? There are some conflicts

I will close this PR and submit the PR directly to OSD main.

**Pull request closed without merging**